pkg/v20/resource/clusterconfigmap/current.go

```diff
@@ -34,6 +34,15 @@ func (r *StateGetter) GetCurrentState(ctx context.Context, obj interface{}) ([]*
 		return nil, microerror.Mask(err)
 	}
 
+	// Cluster namespace is created by the provider operator. If it doesn't
+	// exist yet we should retry in the next reconciliation loop.
+	_, err = r.k8sClient.CoreV1().Namespaces().Get(clusterConfig.ID, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("cluster namespace %#q does not exist", clusterConfig.ID))
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+		resourcecanceledcontext.SetCanceled(ctx)
```

Comment:
I think this should return when canceling.

Reply:
Thanks fixed.

I copied from my old PR and I copied the bug. 🤦‍♂